src/components/Rockets.js

```diff
@@ -0,0 +1,30 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { getRockets } from '../redux/rockets/rockets';
+import Rocket from './Rocket';
+
+function Rockets() {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rocket);
+
+  useEffect(() => {
+    dispatch(getRockets());
+  }, [dispatch]);
```

Comment:
- [ ] Each time this component is rendered, the `getRockets()` action is dispatched. What this means is that there's an API call every time this component renders, thereby fetching new Rockets data. As a result of this, if I reserve a rocket, leave the rockets page, and come back to it, my reserved rockets data is lost because the data is now fresh data.
In order to avoid this and to ensure that your rockets data is fetched only once, please dispatch the `getRockets()` action only once in your `App` component 😎

Reply:
@codepantha  fixed